src/components/response/ResponseBlock.tsx

```diff
@@ -140,9 +141,8 @@ export default function ResponseBlock({
   return (
     <div style={style}>
       {responses.map((response) => (
-        <>
+        <React.Fragment key={`${response.id}-${id}`}>
```

Comment:
Does this stop some re-rendering?

Reply:
Theoretically yes, it could, but really it's just to get rid of the console error which was saying theres duplicate keys in a list, since the fragment is technically the part of the list which the key is actually checked on since its the top element